src/shared/Log.h

```diff
@@ -52,15 +52,13 @@ enum LogFilters
     LOG_FILTER_COMBAT             = 0x000800,               // 11 attack states/roll attack results/etc
     LOG_FILTER_SPELL_CAST         = 0x001000,               // 12 spell cast/aura apply/spell proc events
     LOG_FILTER_DB_STRICTED_CHECK  = 0x002000,               // 13 stricted DB data checks output (with possible false reports) for DB devs
-    LOG_FILTER_AHBOT_SELLER       = 0x004000,               // 14 Auction House Bot seller part
-    LOG_FILTER_AHBOT_BUYER        = 0x008000,               // 15 Auction House Bot buyer part
-    LOG_FILTER_PATHFINDING        = 0x010000,               // 16 Pathfinding
-    LOG_FILTER_MAP_LOADING        = 0x020000,               // 17 Map loading/unloading (MAP, VMAPS, MMAP)
-    LOG_FILTER_EVENT_AI_DEV       = 0x040000,               // 18 Event AI actions
-    LOG_FILTER_DB_SCRIPT          = 0x080000,               // 19 Db scripts
+    LOG_FILTER_PATHFINDING        = 0x010000,               // 14 Pathfinding
```

Comment:
I'd like to have @cyberium or @killerwife opinion here about the change in the way logs are indexed. Changing the indexes on the right without changing the enum on the left can be misleading, in my opinion. If the enums were left untouched on purpose, then the indexing should not be changed and the removed ones should be marked as free for re-use.

Reply:
Given that there already is `// reserved for future version` for a "missing" enum value I should've done similar for those I removed and not messed with the index numbers (not even sure why I did that).
I'll fix this when I can spare the time. Good catch.

Reply:
Might even be an error here. Noticed just now that LOG_FILTER_COUNT seems to include the skipped 3rd index. I skipped two additional indexes, should then LOG_FILTER_COUNT remain 20?
I'll have to dig deeper into the code to understand how this constant is used.

Reply:
Aside from this possible logging issue, the PR seems fine to me and I shall merge it once the logs are sorted out. 👍

Reply:
That's great, thanks for taking the time!
I think I might be able to look at the log issue after work (in 4-5 hours). I'll get back to you then.

Reply:
Looking through the code I don't think there was an actual error, but as noted it was misleading to rearrange the indexes like I did.
I've now added some empty entries for the freed log filters so the slots can be reused in the future.